scripts/check_frontend_test_coverage.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Check for decrease in coverage from 100% of frontend files."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import shutil
+import subprocess
+import sys
+
+import python_utils
+
+LCOV_FILE_PATH = '../karma_coverage_reports/lcov.info'
+DEV_LCOV_FILE_PATH = './tmp/dev-lcov.info'
+PR_LCOV_FILE_PATH = './tmp/pr-lcov.info'
+
+
+def change_git_branch(branch):
+    """Changes git branch.
+
+    Args:
+      branch: string. Name of the branch to be changed.
```

Comment:
Here and elsewhere: use 4-space indentation; use correct typeinfo (it should be what you get when you call type() on the argument).

@kevinlee12 -- these should really be lint checks. I've noticed errors here keep happening; can we bump the priority of these ones up a bit?

Reply:
The indentation check here isn't possible using pylint out of the box since this is inside of a docstring. That being said, I've created https://github.com/oppia/oppia/issues/8165.